src/client/dfs/dfs.c

```diff
@@ -1223,7 +1236,7 @@ dfs_mount(daos_handle_t poh, daos_handle_t coh, int flags, dfs_t **_dfs)
 	dfs->root.parent_oid.lo = RESERVED_LO;
 	dfs->root.parent_oid.hi = SB_HI;
 	daos_obj_generate_id(&dfs->root.parent_oid, 0, OC_RP_XSF, 0);
-	rc = open_dir(dfs, DAOS_TX_NONE, dfs->super_oh, amode, 0, &dfs->root);
+	rc = open_dir(dfs, DAOS_TX_NONE, dfs->super_oh, amode, 0, &dfs->root, 1);
```

Comment:
(style)  line over 80 characters

Reply:
i hate checkpatch errors, so please fix this one too ;-)

Reply:
ah looks like outdated / already fixed